Various editorial tweaks. (#34)
richsalz/draft-ietf-uta-rfc6125bis: draft-ietf-uta-rfc6125bis.md

layer match the TLS-layer.

At the time of this writing, other protocols such as {{QUIC}} and

> Network Time Security ({{NTS}}) use DTLS or TLS as a service to do the

Author: Good point, done.

  service such as a specific resource (via the URI "path" component) or
  parameters (via the URI "query" component).

* CA policies.
  This includes items such as the following:

>   * How to certify or validate FQDNs and application

Author: 6125 says "relative domain names". To avoid the ambiguity I changed it to "or other".

for use in this specification.

application service:
: A service on the Internet that enables clients to connect for the
  purpose of retrieving or uploading information, communicating with other
  entities, or connecting to a broader network of services.

application service provider:
: An entity that hosts or deploys an application service.

application service type:
: A formal identifier for the application protocol used to provide a
  particular kind of application service at a domain.  This often appears as

>   a URI scheme {{URI}} or a DNS SRV Service {{DNS-SRV}}.

Reviewer: ALPN may belong here too.

Reviewer: In some scenarios ALPN is the "formal identifier for an application protocol".  I need to re-read the full text to see if this was just a stupid comment or something that makes sense. The "may" in my statement indicates that I was unsure. :-)

Reviewer: `a URI scheme {{URI}}, a DNS SRV Service {{DNS-SRV}} or an ALPN {{ALPN}} identifier.`

I think that's a better definition.

Author: Agreed, thanks!

through DNS resolution of the source domain.  This rule is important because
only a match between the user inputs and a presented identifier enables the
client to be sure that the certificate can legitimately be used to secure the

> client's communication with the server. Put another way, it also
> removes DNS from the attack surface.

Reviewer 2: ```suggestion
client's communication with the server. This removes
DNS and DNS resolution from the attack surface.
```

It's a suggestion. It feels a bit colloquial, but it could be entirely fine as-is, so feel free to reject :)

## Preparing to Seek a Match {#verify-seek}

Once the client has constructed its list of reference identifiers and has

> received the server's presented identifiers,

Reviewer 2: Did you also want to update (new) lines 632/633?

I was originally surprised by this edit, but with the overall simplifications for CN handling, and this definition is unambiguously present, I like this approach you're taking here. I could see an argument for 632/633 being similarly redundant, or I could see the argument for making sure to restate (during the high level) that these come from the certificate.

Just wanted to flag
